recipe/meta.yaml

```diff
@@ -43,8 +43,10 @@ outputs:
       host:
         - libgomp      # [linux]
         - llvm-openmp  # [osx]
+        - cuda-version {{ cuda_compiler_version }}  # [(cuda_compiler_version or "None") != "None"]
       run:
         - __cuda  # [cuda_compiler_version != "None"]
+        - cuda-version {{ cuda_compiler_version }}  # [(cuda_compiler_version or "None") != "None"]
```

Comment:
Note: The `cuda-version` changes were included in PR ( https://github.com/conda-forge/parallelproj-feedstock/pull/75 ). Including relevant code snippet below for reference

https://github.com/conda-forge/parallelproj-feedstock/blob/2e4d60cdf4fb0f18f2073dba5f665df6324488c8/recipe/meta.yaml#L44-L50